appinfo/info.xml

```diff
@@ -18,7 +18,7 @@
 	<screenshot>https://raw.githubusercontent.com/nextcloud/twofactor_totp/dd1e48deec73a250886f35f3924186f5357f4c5f/screenshots/settings.png</screenshot>
 	<dependencies>
 		<php min-version="7.3" max-version="8.0" />
```

Comment:
needs fixing for full 24 + 25 support?

Reply:
I don't tihnk so. It shows what the app is compatible with.
Whatever server is is irrelevant here